sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs

```diff
@@ -32,19 +33,26 @@ internal RecognizedForm(DocumentResult_internal documentResult, IReadOnlyList<Pa
         }
 
         /// <summary>
+        /// The type of form the model identified the submitted form to be.
         /// </summary>
         // Convert clusterId to a string (ex. "FormType1").
         public string FormType { get; }
 
         /// <summary>
+        /// The range of pages this form spans.
         /// </summary>
         public FormPageRange PageRange { get; }
 
         /// <summary>
+        /// A dictionary of the key-value pairs recognized from the input document. Labeled
+        /// models will use pre-defined labels as keys, and a unique key will be generated
+        /// for <see cref="FormField"/> instances with no pre-defined label available.
         /// </summary>
         public IReadOnlyDictionary<string, FormField> Fields { get; }
 
         /// <summary>
+        /// A list of pages describing the recognized layout elements present in the input
```

Comment:
Let's avoid saying layout to be consistent within our own API.

```suggestion
        /// A list of pages describing the recognized form content elements present in the input
```